src/wolframite/core.clj

```diff
@@ -199,7 +204,7 @@
      (let [with-eval-opts (merge {:jlink-instance jlink-inst}
                                  (:opts jlink-inst)
                                  eval-opts)
-           expr' (unqualify (if (string? expr) (express/express expr with-eval-opts) expr))]
```

Comment:
No more automatic unqualify, so that we can actually send qualified names and invoke functions only available in some packages. Also, there is less need for unqualification, since most users should use the w/ functions, that do not produce qualified symbols in our expressions, i.e. there is less need for syntax quote.